lib/pages/home_screen/movie_list_widget.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:the_movie_db_module2_part1/pages/movie_info_screen/app_movie_info.dart';
+import 'package:the_movie_db_module2_part1/pages/home_screen/movie_list_card.dart';
+import 'package:the_movie_db_module2_part1/repository/movie_repository.dart';
+import '../../model/movie_data_model.dart';
+
+class MovieListScreen extends StatefulWidget {
+  const MovieListScreen({super.key});
+
+  @override
+  State<MovieListScreen> createState() => _MovieListScreenState();
+}
+
+class _MovieListScreenState extends State<MovieListScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder(
+        future: MovieRepository.readJsonData(),
+        builder: (context, data) {
+          if (data.hasError) {
+            return Text(
+              '${data.error}',
+            );
+          }
+          else if (data.hasData) {
+            var movies = data.data as List<MovieDataModel>;
+            return ListView.builder(
+                itemCount: movies == null ? 0 : movies.length,
+                shrinkWrap: true,
```

Comment:
Why this flag is true? It may have a significant performance impact with huge lists
https://api.flutter.dev/flutter/widgets/ScrollView/shrinkWrap.html